keuzewijzer-frontend/src/views/Cohort/CohortCreateView.ts

```diff
@@ -72,7 +72,7 @@ export class CohortCreateView implements View {
     const yearError = $('#yearError');
     const userError = $('#userError');
 
-    if (name.length < 4 || name.length > 100) {
+    if (name.length < 4 || name.length > 254) {
       nameError.text('Semester item naam moet tussen de 4 en 100 karakters zijn.');
```

Comment:
Hier moet dan ook 254 komen te staan. En waarom 254? De database heeft varchar 450 toch?

Reply:
Als je bijvoorbeeld bij de AspNetUsers tabel kijkt zie je dat er een aantal velden NVARCHAR(450) zijn en een aantal NVARCHAR(MAX).

Ik zie nu dat de naam van een semesteritem NVARCHAR(MAX) is, wat [meer dan genoeg is](https://stackoverflow.com/questions/11131958/what-is-the-maximum-characters-for-the-nvarcharmax). Er is dus geen database reden om het te limieteren op 254.

Reply:
Er is inderdaad geen database reden, maar dat hoeft ook niet kan ook een functionele reden zijn.
Maakt mij niet perse uit wat de conclusie is, alleen even de nameError.text aanpassen van 100 naar 254 of 450 

`nameError.text('Semester item naam moet tussen de 4 en 100 karakters zijn.');
`

Reply:
zal de tekst aanpasssen naar 254 had 254 voor database reden maar wist niet zeker of dit van toepassing was bij c#